### PR TITLE
handling errors and improved output

### PR DESCRIPTION
**Pull request by CodeDotJS**
A little tweak to make this tool perfect. 

Well, I saw that, even if the system isn't connected to any network, the tool kept running and while stopping the process, it showed errors which kept ruining the beauty of this tool and the code. ( check the screenshot )
Even the uneven gaps didn't look good.

before

So, I tweaked it little to make it even more good, showing messages and handling errors in a better way ( You can compare the screenshots )

after

Well, the downloading speed is quite low here..haha

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -12,9 +13,32 @@ meow(`
...
+
+checkConnection(isConnected => {
+	if (isConnected) {
+		// no need, and even if we show the message, it won't print because of running spinners.
+	} else {
+		console.log(chalk.red('\n Please check you internet connection.\n'));
+		// exiting if the network is unreachable
+		process.exit(1);
+	}
+});
```
Comment (sindresorhus) [inline cli.js]: Too much boilerplate. Just make it:

```js
// check connection
dns.lookup('fast.com', err => {
	if (err && err.code === 'ENOTFOUND') {
		console.log(chalk.red('\n Please check you internet connection.\n'));
		process.exit(1);
	}
});
```

File: cli.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+
```
Comment (sindresorhus) [inline cli.js]: Don't do unrelated changes

Comment (CodeDotJS) [inline cli.js]: Sorry, sublime automatically did that.

File: cli.js (hunk 1 above)
Comment (CodeDotJS) [inline cli.js]: Thanks @sindresorhus 
removed unnecessary lines and made the changes.

**Comment by sindresorhus** [comment]
Thanks :)
